agents/lakeshore425/LS425agent.py

```diff
@@ -0,0 +1,192 @@
+                acquisition after initialization if True.
+        """
+
+        if params is None:
+            params = {}
+
+        auto_acquire = params.get('auto_acquire', False)
+
+        if self.initialized:
+            return True, "Already Initialized Module"
+
+        with self.lock.acquire_timeout(0, job='init') as acquired:
+            if not acquired:
+                self.log.warn("Could not start init because "
+                              "{} is already running".format(self.lock.job))
+                return False, "Could not acquire lock."
+
+            session.set_status('starting')
+
+            self.dev = ls.LakeShore425(self.port)
+            self.log.info(self.dev.IDN())
+            print("Initialized Lakeshore module: {!s}".format(self.dev))
+
+        self.initialized = True
+        # Start data acquisition if requested
+        if auto_acquire:
+            self.agent.start('acq')
+
+        return True, 'Lakeshore module initialized.'
+
+    def start_acq(self, session, params=None):
+        if params is None:
+            params = {}
+
+        f_sample = params.get('sampling_frequency')
+        # If f_sample is None, use value passed to Agent init
+        if f_sample is None:
+            f_sample = self.f_sample
+
+        sleep_time = 1/f_sample - 0.01
+
+        with self.lock.acquire_timeout(0, job='acq') as acquired:
+            if not acquired:
+                self.log.warn("Could not start acq because {} is already running"
+                              .format(self.lock.job))
+                return False, "Could not acquire lock."
+
+            session.set_status('running')
+
+            self.take_data = True
+
+            session.data = {"fields": {}}
+
+            while self.take_data:
+                Bfield = self.dev.getField()
+                current_time = time.time()
+                
+                data = {
+                    'timestamp': current_time,
+                    'block_name': 'Mag field',
+                    'data': {'Bfield':Bfield}
+                }
+
+                self.agent.publish_to_feed('HallSensor', data)
+                session.data.update({'timestamp': current_time})
+                self.agent.feeds['HallSensor'].flush_buffer()
+
+                time.sleep(sleep_time)
+
+        return True, 'Acquisition exited cleanly.'
+
+    def stop_acq(self, session, params=None):
+        """
+        Stops acq process.
+        """
+        self.dev.close()
+        if self.take_data:
+            self.take_data = False
+            return True, 'requested to stop taking data.'
+        else:
+            return False, 'acq is not currently running'
+    
+    def operational_status(self, session, params=None):
+        self.log.info(self.dev.OPST())
+        return True, 'operational status is checked'
+    
+    def zero_calibration(self, session, params=None):
+        self.log.info(self.dev.ZeroCalibration())
```

Comment:
`self.dev.ZeroCalibration()` -> `self.dev.set_zero()`

Also, does this `ZeroCalibration()` method return anything? It doesn't look like it. If that's the case, we don't need the `self.log.info()` wrapper here.